tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -419,6 +420,55 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUISharingInternalUsers/shareWithUsers.feature:360](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L360)
 -   [webUISharingPublicBasic/publicLinkCreate.feature:190](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature#L190)
 
+### [Admin settings feature not implemented yet](https://github.com/owncloud/web/issues/5015)
```

Comment:
This test suite was deleted last week. IMO we do not need to addd these epected-failures.

Reply:
sorry, I was not aware of that.
removed!